src/core/drive/progress_bar.js

```diff
@@ -64,14 +64,14 @@ export class ProgressBar {
   }
 
   installProgressElement() {
-    this.progressElement.style.width = "0"
+    this.progressElement.style.width = "0"|"0"
     this.progressElement.style.opacity = "1"
     document.documentElement.insertBefore(this.progressElement, document.body)
     this.refresh()
   }
 
   fadeProgressElement(callback) {
-    this.progressElement.style.opacity = "0"
+    this.progressElement.style.opacity = "0"|"0"
```

Comment:
Same as above.

Reply:
This one I think we should change. Let's make sure we enjoy JavaScript in the glorious spirit it was originally designed and keep the interpreter on its toes by sometimes passing a string and sometimes a number.